src/inverseRobot.js

```diff
@@ -7,7 +7,21 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const robotValues = Object.values(robot);
+  const unique = Array.from(new Set(robotValues));
+  const invertedRobotEntries = [];
+
+  if (unique.length < robotValues.length) {
+    return null;
+  } else {
+    for (const key in robot) {
+      const newKey = robot[key];
+
+      invertedRobotEntries.push([newKey, key]);
+    }
+
+    return Object.fromEntries(invertedRobotEntries.reverse());
+  }
```

Comment:
You could avoid nesting. The function stops executing after the return statement anyway
```suggestion
  if (unique.length < robotValues.length) {
    return null;
  }
  
  for (const key in robot) {
    const newKey = robot[key];

    invertedRobotEntries.push([newKey, key]);
  }

  return Object.fromEntries(invertedRobotEntries.reverse());

```